### ISSUE TITLE
Feature proposal QR with plausible deniability (Answ. to #32)

### ISSUE THREAD

**Issue by iuvi7** [paritytech/banana_split#121]
Hello! 

_I would like to thanks all team for work on Banan Split at first_

My proposal\qestion is: Can it possible to introduce inside Banana Split (or side project) something i found out recently - a QR-codes with plausible deniability -> 
https://github.com/sunknudsen/blockcrypt

So you can easily generate **_QR with multiply secrets_** and scan them (second, third and more secret bottoms) and depends what password you enter - _your result will be different_

See as: scan QR, enter 123 -> get "hello", scan QR, enter 321 -> get "Password"

It also partly-answers qestion #32 made by @kirushik how to fit inside more info (?)

P.S. Video of how it works can be found here BUT there no published version for Windows, Linux, Mac or HTML for universality:
https://www.youtube.com/watch?v=bary2XghKjw

Greets!

**Comment by burdges** [comment]
The QR code sounds irrelevant here, unless you want extra shards (papers) that recombine to give a different secret, but that's likely to cause insanity. 

If you have a root key, then produce a primary hard derivation or so using a respectable passphrase, and produce some cover hard derivations using different maybe simpler passphrases, or just use the empty derivation path as the cover.

You put real funds on the primary hard derivation path(s), but only ever put chump change on the cover ones, and you make the tx history look legit.

**Comment by iuvi7** [comment]
> Th/e QR code sounds irrelevant here, unless you want extra shards (papers) that recombine to give a different secret

@burdges Exactly. Only **_one QR_** code with multiply secrets (_**See from 14:06**_) -> https://www.youtube.com/watch?v=bary2XghKjw

Unfortunatley - i'm very low at coding skill's, but would like to help with design and translation to other languages if such project emereges at any time 

P.S. about Banana Split fitting more info in one QR-code... Maybe they can carry "chain-reaction" QR's? 

Sounds crazy, but for example: 
+ Scanning QR: "knkj2kjb10djnk#lknlfdsjlfnjlds" will give birth to "knkj2kjb10djnk" and "lknlfdsjlfnjlds" due splitting symbol # in between to use them further seperatley? (As a "Mother" $ two "kid's")

+ Or something like: Entering _**HMAC**_ for calculating different Hash (Checksum) as it _**birth different result**_ (like in HashCalculators) and transformation of them with such dependency?

**Comment by burdges** [comment]
There is absolutely no reason to make the QR codes contain more since the password itself contains different entropy already.  This is already baked into every crypto currency accounts system via hard derivations. 

The only question is when & how do you have authentication tags that tell the use they enteres the wrong or right thing.  Hard derivations do this with the on-chain data, but other options exist.
